### ISSUE TITLE
Memory leak

### ISSUE THREAD

**Issue by jnckjian** [RastislavMirek/FlexColorPicker#37]
in ColorPickerThumbView
var delegate: LimitedGestureViewDelegate? 
need add weak
weak var delegate: LimitedGestureViewDelegate?

**Comment by RastislavMirek** [comment]
That is correct. Thank you for the report. I can fix that quickly but release of new CocoaPods version will not be so fast...
